image/image_linux.go

```diff
@@ -890,7 +889,9 @@ var setupSeed = func(tsto *tooling.ToolingStore, model *asserts.Model, opts *Opt
 				return nil, err
 			}
 			if !sn.Info.Revision.Unset() {
-				imageManifest[sn.Info.SnapName()] = sn.Info.Revision
+				if err := s.manifest.MarkSnapRevisionSeeded(sn.Info.SnapName(), sn.Info.Revision.N); err != nil {
```

Comment:
but not here, we should pick one approach, also we need to think whether the error is understable and whether we are reapting the snap name too much or not

Reply:
I wrapped the error, and make sure we don't mention the snap name twice as it is indeed a bit too much